admin_manual/enterprise_clients/publishing_android_app.rst

```diff
@@ -0,0 +1,333 @@
+=======================================================
+Distributing Your Branded Android App (Enterprise Only)
+=======================================================
+
+Now that you have created your branded Android app with ownCloud's ownBuilder 
+service (:doc:`creating_branded_apps`) how do you distribute it to your users? 
+There are multiple ways: :ref:`email`, :ref:`publish_server`, or 
+:ref:`publish_google_play`.
+
+
+.. _email:
+
+Email
+-----
+
+You can download your branded Android app from your account on 
+`Customer.owncloud.com <https://customer.owncloud.com/owncloud>`_, and send it 
+as an email attachment to your users. When they open your email on their 
+Android phone or tablet, they must first click the the download arrow (bottom 
+left of the screenshot) to download your app.
+
+.. image:: ../images/android_custom_1.png
+
+When the arrow changes to a green checkbox, it has been downloaded. 
+
+.. image:: ../images/android_custom_2.png
+
+Now your user must click on the green checkbox, and this launches the app 
+installer, and all they have to do is follow the installation wizard to install 
+your branded app.
+
+.. image:: ../images/android_custom_3.png
+
+When the installation is complete, the `ownCloud Android App Manual 
+<https://doc.owncloud.com/android/>`_ contains instructions for using the app.
+
+.. _publish_server:
+
+Publish On Your ownCloud Server
+---------------------------------
```

Comment:
They may share the link to the apk, there is no need for them to access to the server